src/auro-icon.js

```diff
@@ -8,8 +8,19 @@ import { html, css } from "lit-element";
 import { classMap } from 'lit-html/directives/class-map';
 import AuroElement from '@alaskaairux/orion-web-core-style-sheets/dist/auroElement/auroElement';
 import penguin from '@alaskaairux/icons/dist/icons/interface/penguin_es6.js';
+import alaska from '@alaskaairux/icons/dist/restricted/AS_es6.js';
+import alaskaTagline from '@alaskaairux/icons/dist/restricted/AS-tagline_es6.js';
 import cacheFetch from './cacheFetch';
 
+import as400 from '@alaskaairux/icons/dist/restricted/AS-400_es6.js';
```

Comment:
How does this impact bundle size? Looking at unpkg, each AS variant seems to be ~4KB. This means anyone using auro-icon will add a total of 4KB * 10 new imports = 40KB to their bundle size.

This impacts downstream components like auro-carousel that also use auro-icon. TBH, that's too much of a performance hit for a very specific use case. How can we optimize this?

Reply:
I feel pretty strongly that airline logos (not just AS, but partners too) are a common enough use case on the floor to warrant design system support for not just AS but all partner airlines. Multiple teams will solve this problem differently.

Reply:
So there are multiple things we can do here. But for the sake of argument, this release goes out as is with planned refactoring

1. I can work with design to get smaller versions of the files, they are SVGs, so we can get pretty small and still get what we want out of this update
1. There are three CEs
  * auro-icon (pure CDN icon support)
  * auro-logo (pure CDN logo support)
  * auro-alaska (embedded Alaska logos)

Keep in mind, the use case for embedding the Alaska logo is response time as it stands right now. There is a strong possibility once we get the CDN problem solved that this CE will move from it being bundled to being served from the CDN. Keeping that in mind, I don't think we even need to address any of these issues as that will be solved?

Reply:
@gusnaughton, I'm not objecting to supporting airline logos in the design system. I think of auro-icon as an easy, lightweight way to embed an icon on a page. If we have to keep adding additional imports and code just to handle the Alaska (or other airline) logos, I think that's an indication that we need a separate component for that use case. With this update, components like auro-carousel will be bringing in the extra logos and code (40KB worth!) to simply display left and right toggles. That doesn't make sense to me. It would be much more efficient in that case for components using auro-icon to simply import the raw SVG (and quicker, since there's no reason to call a CDN).

If we need to get this PR out, then we need to get it out. However, it should be a high priority to refactor the AS logo stuff into its own component for the reasons I outlined above.

@blackfalcon, I think the CEs you described make sense. There might even be an opportunity to share code.